core/src/main/java/io/dekorate/Session.java

```diff
@@ -202,12 +212,17 @@ public void addListener(SessionListener listener) {
       LOGGER.info("Generating manifests.");
       closed.set(true);
       populateFallbackConfig();
+      populateInitialConfig();
```

Comment:
I'd rather pickup the extenral configuration, during Session initalization.
To be fair, it won't make much of a difference (as it will still run before all handlers), still it seems somehow cleaner.

Also, it should definitely run before adding fallback config.

Reply:
After checking alternatives, existing manifests must be read in this method so that AnnotationProcessors in charge of setting input and output directories have already been triggered. i.e. It's not possible to read these existing manifests during session initialization as input directory will still not be declared.

I will move it before `populateFeedbackConfig` and rename it to `readExistingResources` (previous name was devised from a former iteration and doesn't make much sense)